config/crd/bases/api-gateway.consul.hashicorp.com_gatewayclassconfigs.yaml

```diff
@@ -73,6 +73,10 @@ spec:
                       namespace:
                         description: The Consul namespace to use for authentication.
                         type: string
+                      podSecurityPolicy:
```

Comment:
I'm not a huge fan of having this named explicitly after `podSecurityPolicy` since it's already going away w/ K8s 1.25; however, the replacement for it is different enough that I don't think it makes sense to try and munge the concepts together into one field. Thoughts?